src/coinbase/wallet.ts

```diff
@@ -35,7 +36,7 @@ export class Wallet {
 
   private master?: HDKey;
   private seed?: string;
-  private addresses: Address[] = [];
+  private addresses: DeveloperAddress[] = [];
```

Comment:
@alex-stone it can be `DeveloperAddress` or `ExternalAddress`, right?

Reply:
Must be `WalletAddress`